src/schemas/evaluation_schema.py

```diff
@@ -125,14 +125,70 @@
-                "verdict_colors": {
+                "draw_question_verdicts": {
                     "type": "object",
                     "additionalProperties": False,
+                    "required": ["enabled"],
                     "properties": {
-                        "correct": {"type": "string"},
-                        "incorrect": {"type": "string"},
-                        "unmarked": {"type": "string"},
+                        "enabled": {"type": "boolean"},
+                        "verdict_colors": {
+                            "type": "object",
+                            "additionalProperties": False,
+                            "required": ["correct", "neutral", "negative", "bonus"],
+                            "properties": {
+                                "correct": {"type": "string"},
+                                "neutral": {"type": "string"},
+                                "negative": {"type": "string"},
+                                "bonus": {"type": "string"},
+                            },
+                        },
+                        "verdict_symbol_colors": {
+                            "type": "object",
+                            "additionalProperties": False,
+                            "required": ["positive", "neutral", "negative", "bonus"],
+                            "properties": {
+                                "positive": {"type": "string"},
+                                "neutral": {"type": "string"},
+                                "negative": {"type": "string"},
+                                "bonus": {"type": "string"},
+                            },
+                        },
+                        "draw_answer_groups": {
+                            "type": "object",
+                            "additionalProperties": False,
+                            "required": ["enabled"],
+                            "properties": {
+                                "enabled": {"type": "boolean"},
+                                "color_sequence": {**ARRAY_OF_STRINGS,"minItems": 4,"maxItems": 4}
+                            },
+                            "allOf": [
+                                {
+                                    "if": {"properties": {"enabled": {"const": True}}},
+                                    "then": {
+                                        "required": ["color_sequence"],
+                                    },
+                                }
+                            ],
+                        },
                     },
+                    "allOf": [
+                        {
+                            "if": {"properties": {"enabled": {"const": True}}},
+                            "then": {
+                                "required": [
+                                    "verdict_colors",
+                                    "verdict_symbol_colors",
+                                    "draw_answer_groups",
```

Comment:
draw_answer_groups should not be a required property I guess.. We're providing default enabled value too